unfoldingWord/translationCore
# Added ability to edit target language bible in resources reducer

#### This pull request addresses:
This PR updates the verse for the target language bible in the resources reducer.
#1106 


#### How to test this pull request:
Edit a verse and see how when clicking save changes it will update in the target language area,


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1124)
